inc/media_offload.php

```diff
@@ -845,16 +947,26 @@ public function generate_image_meta( $meta, $attachment_id ) {
 		$generate_url_response = $request->call_upload_api( $original_url );
 
 		if ( is_wp_error( $generate_url_response ) || wp_remote_retrieve_response_code( $generate_url_response ) !== 200 ) {
+			if ( OPTML_DEBUG ) {
+				do_action( 'optml_log', "\n call to signed url error\n" );
+			}
 			update_post_meta( $attachment_id, 'optimole_offload_error', 'true' );
 			return $meta;
 		}
 		$decoded_response = json_decode( $generate_url_response['body'], true );
 
 		if ( ! isset( $decoded_response['tableId'] ) || ! isset( $decoded_response['uploadUrl'] ) ) {
+			if ( OPTML_DEBUG ) {
+				do_action( 'optml_log', "\n missing table id or upload url\n" );
```

Comment:
we should avoid adding line delimiters in the log and properly process the message in the capture function, i.e add_log("\n" . $message);